CHANGELOG.md

```diff
@@ -5,6 +5,30 @@ All notable changes to the Adapt authoring tool are documented in this file.
 **IMPORTANT**: For information on how to **correctly and safely** update your installation, please consult **INSTALL.md**.<br/>
 _Note that we adhere to the [semantic versioning](http://semver.org/) scheme for release numbering._
 
+## [0.6.1] - 2018-11-29
+
+Bugfix release primarily to resolve issues introduced in v0.6.0.
+
+### Fixed
+- Copy function fails if course has orphaned components ([#1701](https://github.com/adaptlearning/adapt_authoring/issues/1701))
```

Comment:
Just to point out that the issue link can be simplifed to just `(#1701)`. I don't know how many people would render/preview the markdown outside of GitHub.